pybaseball/teamid_lookup.py

```diff
@@ -33,6 +33,26 @@ def team_ids(season: Optional[int] = None, league: str = 'ALL') -> pd.DataFrame:
     return fg_team_data
 
 
+def mlb_com_team_id(teamName = None):
```

Comment:
Broader point - I think it might be nice to merge this onto the existing `team_ids` function rather than create a new one. Curious what you think (and other devs). 

I think you'd need to add another column for `teamID : team_name`, but then you could merge that onto the dataset produced by that existing function rather than adding a new one.

Reply:
I'd rather creating a new function (as it is now in this Pull Request) instead of adding this new to the existing one because in that way we can keep things modularity isolated and remember that the less arguments the function needs is better according to Uncle Bob on `Clean Code`